envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/containers/HttpBinContainer.kt

```diff
@@ -0,0 +1,24 @@
+package pl.allegro.tech.servicemesh.envoycontrol.config.containers
+
+import org.testcontainers.containers.wait.strategy.Wait
+import pl.allegro.tech.servicemesh.envoycontrol.config.BaseEnvoyTest
+import pl.allegro.tech.servicemesh.envoycontrol.testcontainers.GenericContainer
+import java.util.UUID
+
+class HttpBinContainer : GenericContainer<HttpBinContainer>("kennethreitz/httpbin:latest") {
```

Comment:
This will be ours 3rh echo container. Can't we just use 1? Personally I like this one :) I trust Postman more than some random containers from github.

Reply:
I agree, @Automaat can you explain why EchoContainer is not sufficient in this case?

Reply:
So basically I've added another container because EchoContainer simply returns HTML with a predefined value. In this test, I have to check what headers are passed in request to this container and I don't think I can do it with EchoContainer

Reply:
Fair enough, I didn't know EchoContainer is that much limited. But our HttpsEchoContainer has the features you need. Its better to reuse it than adding new image. And I would move container definition from EnvoyTestConfiguration to your test, to not start it unecessary for every test (just like in EnvoySANValidationTest for example). This is a output you will receive from the container: https://hub.docker.com/r/mendhak/http-https-echo

Reply:
Oh I didn't realize earlier that there are two different echo containers. I will switch to using this HttpsEchoContainer